av/enum.pyi

```diff
@@ -18,17 +29,17 @@ class EnumItem:
 
     def __int__(self) -> int: ...
     def __hash__(self) -> int: ...
-    def __reduce__(self): ...
-    def __eq__(self, other) -> bool: ...
-    def __ne__(self, other) -> bool: ...
+    def __reduce__(self) -> tuple[bool, ...]: ...
+    def __eq__(self, other: object) -> bool: ...
+    def __ne__(self, other: object) -> bool: ...
 
 class EnumFlag(EnumItem):
     flags: tuple[EnumFlag]
 
-    def __and__(self, other): ...
-    def __or__(self, other): ...
-    def __xor__(self, other): ...
-    def __invert__(self): ...
+    def __and__(self, other: object) -> bool: ...
+    def __or__(self, other: object) -> bool: ...
+    def __xor__(self, other: object) -> bool: ...
+    def __invert__(self) -> bool: ...
```

Comment:
Actually, the return type should be EnumFlag, not bool. Looking at test_enums.py

Reply:
I think you are right